.gitignore

```diff
@@ -5,3 +5,4 @@ cabal.sandbox.config
 hscope.out
 codex.tags
 .anvil
+.stack-work
```

Comment:
Yeah, it install all the generated binaries, logs and local packages (the ones that are not included in stackage snapshot). The plus side is that one does not need a sandbox to develop with stack :D